clientUtils/formatValue.test.ts

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/env jest
+import { formatValue, TickFormattingOptions } from "./formatValue"
+
+describe(formatValue, () => {
+    // prettier-ignore
+    const cases: [string, number, string, TickFormattingOptions][] = [
+        ["default", 1, "1", {}],
+        ["default negative", -1, "-1", {}],
+        ["default large", 1000000000, "1 billion", {}],
+        ["default large specific", 1234567890, "1.23 billion", {}],
+        ["default large specific with rounding", 1239999999, "1.24 billion", {}],
+        ["default small", 0.0000000001, "<0.01", {}],
+        ["2 decimals with integer", 1, "1", { numDecimalPlaces: 2 }],
+        ["2 decimals with float", 1.123, "1.12", { numDecimalPlaces: 2 }],
+        ["4 decimals with float", 1.123, "1.123", { numDecimalPlaces: 4 }],
+        ["with unit", 1, "$1", { unit: "$" }],
+        ["negative with unit", -1, "-$1", { unit: "$" }],
+        ["noTrailingZeroes false", 1.10, "1.1", { noTrailingZeroes: true }], 
+        ["noTrailingZeroes true", 1.10, "1.10", { noTrailingZeroes: false }], 
+        ["$ noSpaceUnit true", 1.1, "$1.1", { noSpaceUnit: true, unit: "$" }],
+        ["$ noSpaceUnit false", 1.1, "$1.1", { noSpaceUnit: false, unit: "$" }],
+        ["% noSpaceUnit false", 1.1, "1.1 %", { noSpaceUnit: false, unit: "%" }],
+        ["% noSpaceUnit false", 1.1, "1.1%", { noSpaceUnit: true, unit: "%" }],
+        ["numberPrefixes true", 1000000000, "1 billion", { numberPrefixes: true }],
+        // this case appears to not currently work
+        // ["numberPrefixes false", 1000000000, "1,000,000,000", { numberPrefixes: false }],
+        ["shortNumberPrefixes true", 1000000000, "1B", { shortNumberPrefixes: true }],
+        ["shortNumberPrefixes false", 1000000000, "1,000,000,000", { shortNumberPrefixes: false }],
+        ["showPlus true", 1, "+1", { showPlus: true }],
+        ["showPlus false", 1, "1", { showPlus: false }],
+        ["showPlus false with negative number", -1, "-1", { showPlus: false }],
+        ["showPlus true with unit", 1, "+$1", { showPlus: true, unit: "$" }],
+        ["showPlus true with % and 4 decimals", 1.23456, "+1.2346%", {showPlus: true, numDecimalPlaces: 4, unit: "%"}],
+        ["showPlus false with $ and noTrailingZeroes true", 1234.5678, "$1,234.57", {showPlus: false, unit: "$", noTrailingZeroes: true}],
+        ["showPlus false with $, noTrailingZeroes false, and noSpaceUnit false", 1234.5678, "$1,234.57", {showPlus: false, unit: "$", noTrailingZeroes: false, noSpaceUnit: false}],
+    ]
```

Comment:
Super nice to have these test cases 🙌 

We should try to reproduce the one that is going wrong in the svgTester? I think it might be a combination of number prefixes (due to a large number) in combination with a currency, but not sure.